docs/Configuration/Sonarqube.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Sonarqube"
+sidebar_position: 7
+description: Config UI instruction for Sonarqube
+---
+
+Visit config-ui: `http://localhost:4000`.
+
+### Step 1 - Add Data Connections
+
+![sonarqube-add-data-connections](/img/ConfigUI/sonarqube-add-data-connections.png)
+
+#### Connection Name
+
+Name your connection.
+
+#### Endpoint URL
+
+This should be a valid REST API endpoint
+
+- `https://YOUR_DOMAIN:YOUR_PORT/api/`
+
+The endpoint url should end with `/`.
+
+#### Auth Token(s)
+
+Sonarqube personal access tokens are required to add a connection. Learn about [how to create a Sonarqube personal access token](https://sonarqube.inria.fr/sonarqube/documentation/user-guide/user-token/).
```

Comment:
1. SonarQube personal access tokens
2. Are there requirements for token permissions?

Reply:
I created a token, but I really didn't find a place with permission control